### PR TITLE
Feat: "following-followers"

### PR DESCRIPTION
**Pull request by sadathshariff**
Implemented
Followers
Following
Suggested Users
Navigate to Other person Profile Page
Changes  in
 - Bookmark
 - Filters
and Minor Fixes

### CODE REVIEW

**Review by nishasen — APPROVED** [review]
Everything looks good

File: src/pages/Home/Home.jsx [hunk 1]
```diff
@@ -10,17 +10,28 @@ import {
...
+  // const filterByDate = [...filterPosts].sort(
+  //   (a, b) =>
+  //     new Date(a.data?.timeStamp.seconds * 1000) -
+  //     new Date(b.data?.timeStamp.seconds * 1000)
+  // );
+
+  // const isUndefined = (o) => typeof o.popularity === "undefined";
+  // const filterByLikes = filterPosts?.sort(
+  //   (a, b) => b?.data?.likes?.length - a?.data?.likes?.length
+  // );
```
Comment (aishwarya-bm) [inline src/pages/Home/Home.jsx]: Remove comments if unnecessary

File: src/components/FollowBar/FollowBar.jsx [hunk 2]
```diff
@@ -1,51 +1,42 @@
...
         <h3>Suggestions</h3>
         <ul className="list">
-          {people?.map(({ id, name }) => (
-            <li key={id} className={`${styles.user_info}`}>
-              <Avatar alt={name} />
-              <p>{name}</p>
-              <Button variant="contained" >
+          {trimPeople?.map((peer) => (
+            <li key={peer.id} className={`${styles.user_info}`}>
+              <Avatar alt={peer?.data?.userName} />
```
Comment (aishwarya-bm) [inline src/components/FollowBar/FollowBar.jsx:35]: Are you missing the src attribute here?

File: src/firebaseMethods.js [hunk 3]
```diff
@@ -79,3 +82,75 @@ export const deletePost = async (postId, dispatch) => {
...
+    dispatch(getLoggedInUserData(id));
+    dispatch(getAllUsers());
+    dispatch(
+      openToast({
+        message: `you started following ${peerData?.userName}`,
+        type: "success",
+      })
+    );
+  } catch (error) {
+    console.log("Error Following", error);
```
Comment (aishwarya-bm) [inline src/firebaseMethods.js:122]: Remove console logs

**Review by aishwarya-bm — APPROVED** [review]
Code looks readable and clean.
Avatar tags are missing src attribute, check it out once. Everything else looks good. Like how you are handling the suggestions logic.

File: src/components/FollowBar/FollowBar.jsx (hunk 2 above)
Comment (sadathshariff) [reply src/components/FollowBar/FollowBar.jsx:35]: Yeah left it intentionally, working on image part

File: src/pages/Home/Home.jsx (hunk 1 above)
Comment (sadathshariff) [reply src/pages/Home/Home.jsx]: Thanks for pointing it out, will remove it

**Pull request merged**